views_euid/index.hbs

```diff
@@ -237,6 +232,10 @@
           </button>
         </div>
       </form>
+      <div class="this-site--net-160765-C61RwL raleway-medium-stratos-12px">
+        {{__ "For more information on how the Trade Desk manages EUID and your rights with respect to EUID, please read the "}}
+        <a href="/privacy">{{__ "EUID Privacy Notice"}}</a>.
```

Comment:
Should there be a link title or any other common properties for a link?

Reply:
Not here - the link tag contents provides information about where it will take you.

If you have something like `<a href="...">(Read more...)</a>` or just an image, then you should provide a title for accessibility, but the preferred option is to put it all in the tag content like this.